pyart/graph/radardisplay.py

```diff
@@ -708,7 +774,19 @@ def plot_colorbar(self, mappable=None, field=None, label=None, cax=None,
             if field is None:
                 field = self.plot_vars[-1]
             label = self._get_colorbar_label(field)
-        cb = fig.colorbar(mappable, ax=ax, cax=cax)
+
+        if orient is None:
+            orient = 'vertical'
+
+        # Find the axes locations to set colorbar
+        if cax is None:
```

Comment:
Testing, I found out that when not using `make_axes_locatable` we can't delete the colorbar in artview when the plot size changes, that is probably the reason we are using `colorbar_flag=False`. Unfortunately even with  `make_axes_locatable` there is a problem when changing the field. As this is the case, I will follow @jjhelmus and let matplotlib work with that.